tasks/development.ts

```diff
@@ -208,4 +224,165 @@ export function load() {
     await run(ethsAddress, wrbtcAddress, "eth", "btc");
     await run(rusdtAddress, wrbtcAddress, "usd", "btc");
   });
+
+  task("dev:deploy-timelock")
+    .addParam("proposer", "proposer address")
+    .addParam("owner", "destinination owner address (usually multi-sig)")
+    .setAction(async ({ proposer, owner }, hre) => {
+      const timeLockFactory = await hre.ethers.getContractFactory("Timelock");
+      const deployer = await getDeployer(hre);
+
+      const day = 3600 * 24;
+
+      const timelock = (await timeLockFactory.connect(deployer).deploy(
+        proposer,
+        0,
+        30 * day,
+        14 * day,
+        5 * 60 // 5 min initial delay
+      )) as Timelock;
+
+      await timelock.deployed();
+
+      console.log("Timelock deployed to:", timelock.address);
+
+      await (await timelock.transferOwnership(owner)).wait();
+
+      console.log("Timelock proposer:", await timelock.proposer());
+      console.log("Timelock ownership transferred to:", await timelock.owner());
+    });
+
+  task("dev:deploy-flipper")
+    .addParam("owner", "destinination owner address (usually timelock)")
+    .setAction(async ({ owner }, hre) => {
+      const deployer = await getDeployer(hre);
+      const flipperFactory = await hre.ethers.getContractFactory("Flipper");
+      const flipper = (await flipperFactory.connect(deployer).deploy()) as Flipper;
+      await flipper.deployed();
+
+      console.log("Flipper deployed to:", flipper.address);
+
+      await (await flipper.transferOwnership(owner)).wait();
+
+      console.log("Flipper ownership transferred to:", await flipper.owner());
+    });
+
+  task("dev:flipper-flip-0")
+    .addPositionalParam("flipperAddress")
+    .setAction(async ({ flipperAddress }, hre) => {
+      const deployer = await getDeployer(hre);
```

Comment:
If it's dev only, can we add an if condition if it's `mainnetFork` and if not throw an error?